cypress/cypress/integration/home.spec.ts

```diff
@@ -100,17 +100,48 @@ describe('Review answers page', () => {
       });
     });
   });
-  it('shows mentor scope card', () => {
+  it('shows mentor info card', () => {
     cySetup(cy);
     cyMockDefault(cy, { mentor: clint });
     cy.visit('/');
+    cy.get('[data-cy=stage-card]').contains('Clinton Anderson');
+    cy.get('[data-cy=stage-card]').contains('Video Mentor');
+    cy.get('[data-cy=stage-card]').contains(
+      "Title: Nuclear Electrician's Mate"
+    );
+    cy.get('[data-cy=stage-card]').contains('Last Trained: Today');
     cy.get('[data-cy=stage-card]').contains('Scope: Incomplete');
     cy.get('[data-cy=stage-card]').contains("This Mentor can't be built yet.");
     cy.get('[data-cy=stage-progress]').should('exist');
     cy.get('[data-cy=stage-card]')
       .find('[data-cy=next-stage-info]')
       .trigger('mouseover');
     cy.contains('This Mentor can select questions from a list');
+    cy.get('[data-cy=thumbnail-wrapper]').trigger('mouseover');
+    cy.get('[data-cy=upload-file]').should('exist');
+  });
+  it('shows placeholder when no thumbnail', () => {
+    cySetup(cy);
+    clint.thumbnail = '';
+    cyMockDefault(cy, { mentor: clint });
+    cy.visit('/');
+    cy.get('[data-cy=mentor-thumbnail]').should('not.have.attr', 'src');
+  });
+  it('switches to new image when uploaded', () => {
+    cySetup(cy);
+    cyMockDefault(cy, { mentor: clint });
+    cy.visit('/');
+
+    cy.get('[data-cy=thumbnail-wrapper]').trigger('mouseover');
+    cy.fixture('avatar.png').then((fileContent) => {
+      cy.get('input[type="file"]').attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'avatar.png',
+        mimeType: 'avatr.png',
+      });
+      cy.get('[data-cy=mentor-thumbnail]').should('have.attr', 'src');
```

Comment:
this test should prove that client refetched mentor (gql) and that there's now a thumbnail, (i.e. set up the mock for gql to have empty thumbnail on first query and then thumbnail set to something u can test on second)